python-pocketlint.spec

```diff
@@ -34,18 +34,18 @@ Summary: Support for running pylint against projects (Python 2 version)
 %{?python_provide:%python_provide python2-pocketlint}
 
 BuildRequires: python2-devel
-BuildRequires: python-six
-BuildRequires: python-futures
+BuildRequires: python2-six
+BuildRequires: python2-futures
```

Comment:
Is this going to work on RHEL/CentOS 7?

Reply:
Yes, both works on CentOS 7.

```
Description : This package exists only to allow packagers to uniformly depend on
            : python2-six instead of conditionalizing those dependencies based on the
            : version of EPEL or Fedora.  It contains no files.
```